packages/page/src/Application/Message/CopyPageMessage.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) Sulu GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Page\Application\Message;
+
+class CopyPageMessage
+{
+    /**
+     * @param array{
+     *     uuid?: string
+     * } $identifier
+     * @param array{
+     *     uuid?: string
+     * } $destinationIdentifier
+     */
+    public function __construct(private array $identifier, private array $destinationIdentifier)
```

Comment:
```suggestion
    public function __construct(
        private array $identifier,
        private array $destinationParentIdentifier,
        private ?string $destinationUuid = null,
    ) {
```

It should be possible to set the uuid from outside like we allow it in create message also: https://github.com/sulu/sulu/blob/3.0/packages/page/src/Application/Message/CreatePageMessage.php